p4-16/psa/psa.p4

```diff
@@ -524,8 +524,8 @@ extern ActionSelector {
 
 // BEGIN:Digest_extern
 extern Digest<T> {
-  Digest(PortId_t receiver); /// define a digest stream to receiver
-  void emit(in T data);      /// emit data into the stream
+  Digest(PortId_t receiver);    /// define a digest stream to receiver
+  void add_metadata(in T data); /// emit data into the stream
```

Comment:
`void add_metadata<T>(in T data); /// ...`

Reply:
While this might look OK from the data plane side, it makes it fairly difficult to generate proper control plane APIs, that are supposed to somehow extract the data that we put into the Digest receiver in the data plane using the names of the corresponding fields.